nanoFramework.WebServer/nanoFramework.WebServer.nfproj

```diff
@@ -91,6 +91,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
If you're adding this here, need to include the packages.lock.json file to the PR.

Reply:
it is already present as files